README.md

```diff
@@ -1,14 +1,14 @@
 ### Postfix Redis
-This is a patch to the postfix 2.9.3 MTA to add a lookup table for redis database.
+This is a patch to the postfix 3 MTA to add a lookup table for redis database.
```

Comment:
Can we be more specific of the postfix version here and not just major version.

Reply:
ok will have a look into it